src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -284,7 +284,13 @@ internal static HttpRequestMessage BuildSetTriggersRequest()
                 protocol = "http";
             }
 
-            var url = $"{protocol}://{Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHostName)}/operations/settriggers";
+            var hostname = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHostName);
+            // Linux Dedicated on AppService doesn't have WEBSITE_HOSTNAME
```

Comment:
Why isn't this set on Linux Dedicated? As you'll see in my active PR https://github.com/Azure/azure-functions-host/pull/4145 I'm also relying on WEBSITE_HOSTNAME being set to form the base URL in GetBaseUrl() for list function response hrefs.

For the purposes of the sync triggers call what you have will be fine, but for forming hrefs it's ideal if they respect any custom host name

Reply:
Good question. I was under the impression that it was added there as well, but clearly it's not there today. We need to follow up with Ranjith's team to get this added there to reduce random environment changes between the various offerings. 

I need to release this, so I targeted this PR against master. Should I merge it to master, then merge master to dev? or how do you do this?